Test/CourseDashboardViewControllerTests.swift

```
@@ -100,7 +100,7 @@ class CourseDashboardViewControllerTests: SnapshotTestCase {
     func testCertificate() {
         let interface = OEXInterface()
         let courseData = OEXCourse.testData()
-        let enrollment = UserCourseEnrollment(dictionary: ["certificate":["url":"test"], "course" : courseData])
+        let enrollment = UserCourseEnrollment(dictionary: ["certificate":["url":"test"], "course" : courseData])!
```

For tests I feel like we should fail fast. It would be nice if the test didn't *crash* in such a failure, but otherwise we just need awkwardly treat it as optional and potentially not notice if it does get messed up.